### DISCUSSION TITLE
Module compiled with Swift 5.5 cannot be imported by the Swift 5.5.2 compiler

### DISCUSSION THREAD

**Discussion by ghost** [LeoNatan/LNPopupController#500] [General] [answered]
Hi,

after I update Xcode to the latest version available 13.2, I started to see this error **Module compiled with Swift 5.5 cannot be imported by the Swift 5.5.2 compiler**

I tried to do Carthage update and build but I had this problem: 

A shell task (/usr/bin/xcrun xcodebuild -project /Users/user/Documents/footters-ios-swift/Carthage/Checkouts/LNPopupController/LNPopupControllerExample/LNPopupControllerExample.xcodeproj CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES -list) failed with exit code 74:
2021-12-14 16:12:48.544 xcodebuild[3848:21667]  DVTAssertions: Warning in /Library/Caches/com.apple.xbs/Sources/DVTiOSFrameworks/DVTiOSFrameworks-19527/DTDeviceKitBase/DTDKRemoteDeviceData.m:373
Details:  (null) deviceType from 00008027-001559060E01402E was NULL when -platform called.
Object:   <DTDKMobileDeviceToken: 0x7feceda4f100>
Method:   -platform
Thread:   <NSThread: 0x600002ed2c80>{number = 8, name = (null)}
Please file a bug at https://feedbackassistant.apple.com with this warning message and any useful information you can provide.
xcodebuild: error: Could not resolve package dependencies:
  Internal error: missingPackageDescriptionModule

Do you have any idea of how to solve this?

Thanks

**Comment by LeoNatan** [comment, marked as answer]
Sorry, not really familiar with Carthage. Make sure your commandline tools point to the correct Xcode.
